schemas/context/browserdetails.schema.json

```diff
@@ -16,13 +16,23 @@
         "xdm:name": {
           "title": "Name",
           "type": "string",
-          "description": "The application or browser version."
+          "description": "The application or browser name."
+        },
+        "xdm:vendor": {
+          "title": "Vendor",
+          "type": "string",
+          "description": "The application or browser vendor."
         },
         "xdm:version": {
           "title": "Version",
           "type": "string",
           "description": "The application or browser version."
         },
+        "xdm:versionMinor": {
```

Comment:
if you are going to have versionMinor, then the other needs to be versionMajor

Reply:
Removed major/minor distinctions - using simple version field.